docs/data/README.mdx

```diff
@@ -2,10 +2,28 @@
 title: Data Overview
 ---
 
-The RPC, data indexers, Hubble, and Horizon are platforms or tools that facilitate communication between users and applications and the Stellar network. With these various platforms, users can query network data, submit transactions, and interact with smart contracts without needing to understand the low-level details of Stellar Core.
+The RPC, data indexers, analytical platforms, Galexie, Hubble, and Horizon are platforms or tools that facilitate communication between users and applications and the Stellar network. With these various platforms, users can query network data, submit transactions, and interact with smart contracts without needing to understand the low-level details of Stellar Core.
 
 This section will walk you through the differences between the various platforms and tools, what platform or tool is best for what use case, and then link to their various documentation locations.
 
+| Features               | RPC | Horizon | Galexie | Hubble |
+| ---------------------- | --- | ------- | ------- | ------ |
+| Real-time Data         | ✅  | ✅      | ✅      | ❌     |
+| Historical Data        | ❌  | ❌      | ✅      | ✅     |
+| Classic Data           | ✅  | ✅      | ✅      | ✅     |
+| Soroban Data           | ✅  | ❌      | ✅      | ✅     |
+| Transaction Submission | ✅  | ✅      | ❌      | ❌     |
+| Ad Hoc Data Analysis   | ❌  | ❌      | ❌      | ✅     |
+| Dashboarding           | ❌  | ❌      | ❌      | ✅     |
+
+:::note
+
+Data indexers can be implemented with RPC, Horizon, and/or Galexie.
+
+Analytical platforms provide an OLAP style access to data similar to Hubble.
```

Comment:
Do you think we would list analytical platforms here beyond Hubble? This note reads a little awkwardly to me. Can you give more context on what you're trying to convey with it?

Reply:
Originally I wanted to address everything listed in the intro paragraph which included data indexers and analytics platforms. My original intent was to add the note so the user wouldn't be confused why data indexers and analytics platforms weren't in the chart.

BUT with the `Stellar supports tools that facilitate communication between users and applications on the Stellar network` line change, I think I will just remove the note completely

Reply:
updated in updated in a088c83120488cfcf7ef46830ab6621107828e2c